### DISCUSSION TITLE
nodeVersion in future

### DISCUSSION THREAD

**Discussion by keeema** [cypress-io/cypress#18998] [General]
Hello,

please what version of node will be used when nodeVersion parameter will be removed? 

System or bundled?

**Comment by mjhenkes** [comment]
With version 9.0.0 we changed the behaviour of cypress to default to using `system` node. Additionally the behaviour of `system` node was changed to always use the version of node used to launch the cli. When `nodeVersion` is removed cypress will continue to use the node version of the cli that launched it to build files in the integrationFolder, build files in the supportFile, and execute code in the pluginsFile.
